system/Database/BaseBuilder.php

```diff
@@ -1358,9 +1359,9 @@ public function set($key, $value = '', ?bool $escape = null)
             if ($escape) {
                 $bind = $this->setBind($k, $v, $escape);
 
-                $this->QBSet[$this->db->protectIdentifiers($k, false, $escape)] = ":{$bind}:";
+                $this->QBSet[$this->db->protectIdentifiers($k, false)] = ":{$bind}:";
             } else {
-                $this->QBSet[$this->db->protectIdentifiers($k, false, $escape)] = $v;
+                $this->QBSet[$this->db->protectIdentifiers($k, false)] = $v;
```

Comment:
The same as above.

Reply:
This change was merged into CI, so I rebased it.